qiskit/algorithms/gradients/finite_diff_estimator_gradient.py

```diff
@@ -75,7 +75,7 @@ def __init__(
         if epsilon <= 0:
             raise ValueError(f"epsilon ({epsilon}) should be positive.")
         self._epsilon = epsilon
-        self._base_parameter_values_dict = {}
+        self._base_parameter_values_dict = {}  # TODO: unused?
```

Comment:
This parameter appear to not be used anywhere.

Reply:
In that case it seems safe to remove it 👍🏻